assets/app/widgets/suse-lazy-spinner/suse-lazy-spinner.less

```diff
@@ -0,0 +1,8 @@
+/* Start= widgets/suse-lazy-spinner/suse-lazy-spinner.less */
+suse-lazy-spinner {
```

Comment:
Bootstrap has a helper class to hide elements: `.hidden`. Maybe you can use such class to switch between it and `display: inline-block` as a default style.
http://getbootstrap.com/css/#helper-classes-show-hide